test.js

```diff
@@ -0,0 +1 @@
+console.log("TEST SUCCESSFUL");
```

Comment:
Will be replaced with something else soon. This is for continuous integration purposes.